source/Fleet.cpp

```diff
@@ -472,7 +472,7 @@ void Fleet::Place(const System &system, list<shared_ptr<Ship>> &ships, bool carr
 		Point pos = center.first + Angle::Random().Unit() * OffsetFrom(center);
 		double velocity = 0;
```

Comment:
change this to speed if you change the other one to speed

Reply:
done